aws-sagemaker-hosted-scorer/examples/example_request.py

```diff
@@ -0,0 +1,32 @@
+import boto3
+
+client = boto3.client('sagemaker-runtime')
+
+endpoint_name = "endpoint-name"
+content_type = "application/json"
+payload = """
```

Comment:
Optional: We could also use the `json` library here to have a native Python struct converted to Json (see here: https://docs.python.org/3/library/json.html). Feel free to keep. Your call.